client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java

```diff
@@ -56,13 +58,26 @@
  * > public class Foo {
  * >     private int a;
  * >     private String b;
+ * >     private Revision rev;
  * >
  * >     public int getA() { return a; }
  * >     public void setA(int a) { this.a = a; }
  * >     public String getB() { return b; }
  * >     public void setB(String b) { this.b = b; }
+ * >     public Revision getRevision() { return this.rev; }
  * > }
  * }</pre>
+ *
+ * In the following example, callback will be called when new value is updated
+ * <pre>{@code
+ *  CentralDogmaFactory factory = new CentralDogmaFactory(dogma, new ObjectMapper());
+ *  Consumer<Foo> fooUpdatedListener = (Foo f) -> {
+ *     System.out.println("foo has updated to: " + f);
+ *  }
```

Comment:
- (Foo f) -> f
- Missing semicolon after }
- Unnecessary extra single space in every line?

  ```
  *  CentralDogmaFactory .. ->
  * CentralDogmaFactory ..
  ```